projects/SynthIPTemplate/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(Template main.cpp)
+
+target_compile_options(Template PUBLIC)
```

Comment:
If all CS are the same base code, should we not exclude exceptions for all of them?